controllers/articles.controller.js

```diff
@@ -0,0 +1,12 @@
+const { fetchArticleById } = require("../models/fetchArticleById.model.js");
+
+exports.getArticleById = (req, res, next) => {
+  const articleId = req.params.article_id;
+  fetchArticleById(articleId)
+    .then((article) => {
+      res.status(200).send({ article });
```

Comment:
This is an appropriate response being sent on a well titled key of 'article' - well done